telegram/ext/filters.py

```diff
@@ -192,12 +192,126 @@ def filter(self, message):
     class _Document(BaseFilter):
         name = 'Filters.document'
 
+        class category(BaseFilter):
+            """This Filter filters documents by their category in the mime-type attribute
+
+            Note:
+                This Filter only filters by the mime_type of the document,
```

Comment:
You didn't put an `Args:` section, and you should specifically mention that `mime_type.startswith` is used

Reply:
Ah I see you put the `Args:` in the `__init__`, sorry